terraform/variables.tf

```diff
@@ -0,0 +1,94 @@
+variable "additional_user_data_script" {

```

Comment:
this file already exist in the folder.
You should rebase your branch from master...